legend-engine-xt-relationalStore-grammar/src/main/java/org/finos/legend/engine/language/pure/grammar/from/RelationalParseTreeWalker.java

```diff
@@ -399,6 +400,15 @@ else if (ctx.NOT_NULL() != null)
                 }
                 break;
             }
+            case "JSON":
+            {
+                column.type = new Json();
+                if (!ctx.INTEGER().isEmpty())
+                {
+                    throw new EngineException("Column data type JSON does not expect any parameters in declaration", this.walkerSourceInformation.getSourceInformation(ctx.identifier().getStart(), ctx.PAREN_CLOSE() != null ? ctx.PAREN_CLOSE().getSymbol() : ctx.identifier().getStop()), EngineErrorType.PARSER);
+                }
+                break;
+            }
```

Comment:
I had the same thought, so I spoke with Pierre. He recommended that I add the JSON type since he wants eventually to have the relational data types in pure match with what each database provides.